.github/workflows/test.yml

```diff
@@ -46,3 +46,18 @@ jobs:
       - uses: codecov/codecov-action@v2
       - run: (cd stela; npm run test -w @stela/access_copy_attacher)
       - uses: codecov/codecov-action@v2
+
+  verify_docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "18"
+
+      - name: Install Redoc CLI
+        run: npm install -g @redocly/cli
+
+      - name: Generate HTML from OpenAPI spec
+        run: npx @redocly/cli build-docs ./packages/api/docs/src/api.yaml -o docs/index.html
```

Comment:
I'm not sure that we need this; I think the `redocly lint` that we run as part of linting will catch anything that would make a build fail (though the docs don't declare that, it's just what I've seen so far).

Reply:
I'll take this out then!